media/css/core.css

```diff
@@ -621,6 +621,14 @@ p.build-missing { font-size: .8em; color: #9d9a55; margin: 0 0 3px; }
   color: #5a5;
 }
 
+a.notification-action {
+    text-decoration: none;
```

Comment:
This would probably be better floated right. On the left, this feels like more of an icon instead of an action:

![image](https://user-images.githubusercontent.com/1140183/125666193-4670fc60-90b1-44ab-8bc3-ab3d19a71f34.png)